lib/ntbb-ladder.lib.php

```diff
@@ -158,7 +158,7 @@ function getRating(&$user, $create=false) {
 				}
 				$rp = $this->getrp();
 				$ladderdb->query(
-					"INSERT INTO `{$ladderdb->prefix}ladder` (`formatid`,`userid`,`username`,`rptime`) VALUES (?,?,?,?)",
+					"INSERT INTO `{$ladderdb->prefix}ladder` (`formatid`,`userid`,`username`,`rptime`,`rpdata`,`col1`) VALUES (?,?,?,?,'',0)",
```

Comment:
Looks good. Does `col1` also not have a default value?

Reply:
Not with the current schema, no, but it's a `double`, so I could add a default value of 0. I'm still getting the hang of PS database stuff, but if you think adding a default value wouldn't break anything that seems like a good change to me!